README.md

```diff
@@ -1 +1,63 @@
-# better-unit
\ No newline at end of file
+# better-unit
+
+better-unit is a wrapper around the [nodeunit](https://www.npmjs.com/package/nodeunit) module, with the following additions:
+
+* Support for comparing `Map`, `Set`, `WeakMap`, and `WeakSet` using the `deepEqual()` assertion.
```

Comment:
We don't know the scope of modifications that will exist in the future. I'd rather not start documenting individual features for use on their own with unbounded future functionality.